app/assets/stylesheets/profile-preview-card.scss

```diff
@@ -8,9 +8,25 @@
 
   &__content.crayons-dropdown:hover {
     display: block;
+    animation: hoverAppear 500ms;
   }
 
   &__trigger:hover + .profile-preview-card__content.crayons-dropdown {
     display: block;
+    animation: hoverAppear 500ms;
+  }
+
+  @keyframes hoverAppear {
```

Comment:
I've used an animation like this so that we don't have to set the opacity of the dropdown content to 0 in its unhovered state (this would add unnecessary complexity to the dropdown helpers).

To make up for not being able to make use of `transition-delay` I've set the animation to stay at `opacity-0` until the very end of the animation, avoiding a "fade in" effect.

NB: I'm using opacity rather than display because we can't animate the display property 😢